Automattic/wp-calypso
# My Home: Update start learning picture

#### Changes proposed in this Pull Request

* Update the picture of the “Blog like an expert from day one” section to align with https://github.com/Automattic/wp-calypso/issues/58763#issuecomment-995484867.

![image](https://user-images.githubusercontent.com/13596067/148043194-dd9f50e7-d0cf-4f9b-a74a-5cdb65575829.png)

#### Testing instructions



* Go to `/home/<your_site>`
* Check the picture of the “Blog like an expert from day one” section is the latest one.



Related to https://github.com/Automattic/wp-calypso/issues/58763
